style.css

```diff
@@ -0,0 +1,628 @@
+@font-face{
+    font-family: 'Lato';
+    src: url(./assets/fonts/Lato/Lato-Light.ttf) format("truetype");
+    font-weight: 100;
+}
+
+@font-face{
+    font-family: 'Lato';
+    src: url(./assets/fonts/Lato/Lato-Regular.ttf) format("truetype");
+    font-weight: 500;
+}
+
+@font-face{
+    font-family: 'Lato';
+    src: url(./assets/fonts/Lato/Lato-Bold.ttf) format("truetype");
+    font-weight: 700;
+}
+
+@font-face{
+    font-family: 'Lato';
+    src: url(./assets/fonts/Lato/Lato-Black.ttf) format("truetype");
+    font-weight: 900;
+}
+
+@font-face{
+    font-family: 'Maven';
+    src: url(./assets/fonts/Maven_Pro/MavenPro-Regular.ttf) format("truetype")
+}
+
+@font-face {
+    font-family: 'fontello';
+    src: url('./assets/fonts/fontello/fontello.eot?42880131');
+    src: url('./assets/fonts/fontello/fontello.eot?42880131#iefix') format('embedded-opentype'),
+         url('./assets/fonts/fontello/fontello.woff2?42880131') format('woff2'),
+         url('./assets/fonts/fontello/fontello.woff?42880131') format('woff'),
+         url('./assets/fonts/fontello/fontello.ttf?42880131') format('truetype'),
+         url('./assets/fonts/fontello/fontello.svg?42880131#fontello') format('svg');
+    font-weight: normal;
+    font-style: normal;
+  }
+
+*{
+    margin: 0;
+    padding: 0;
+}
+
+.body{
+    background-color: #ffffff;
+}
+
+/* Header */
+
+.header{
+    background-color:#323232;
+    height: 62px;
+    color: #fff;
+    font-family: "Lato", Arial, sans-serif;
+    font-weight: bold;
+    width: 100%;
+    font-size: 10px;
+}
+
+.header__navbar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 1020px;
+    margin: 0 auto;
+    position: relative;
+}
+
+.header__navbar::before,
+.header__navbar::after{
+    content:'';
+    width: 40%;
+    height: 1px;
+    background-color: #e4e9eb;
+    position: absolute;
+    bottom: -9px;
+}
+
+.header__navbar::after{
+    left: 31px;
+}
+
+.header__navbar::before{
+    right: 31px;
+}
+
+.navbar__logo{
+    display: inline-block;
+    padding: 0 12px;
+    position: relative;
+    margin: 10px 12px 0;
+    height: 56px;
+    width: 64px;
+}
+
+.navbar__logo img{
+    z-index: 3;
+}
+
+.navbar__logo::before{
+    content: "";
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 27px 46px 0 46px;
+    border-color: #323232 transparent transparent transparent;
+    position: absolute;
+    left: 0;
+    bottom: -23px;
+    z-index: -2;
+}
+
+.navbar__list{
+    display: flex;
+    align-items: center;
+    list-style: none;
+}
+
+.navbar__item{
+    display: inline-block;
```

Comment:
why do you use display: inline-block;? It is unnecessary

Reply:
use space before `{` in css